### ISSUE TITLE
Will the background image also generate txt？or just abandon?

### ISSUE THREAD

**Issue by ShengyuanTang-GD** [ultralytics/JSON2YOLO#20]

**Comment by glenn-jocher** [comment]
@ShengyuanTang-GD no, *.txt files are not required if there are no labels. See https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data#2-create-labels

**Comment by glenn-jocher** [comment]
Thanks @TangsterX. Background images do not require label txt files. They can be included as images with no matching label file, or with an empty label file depending on the training pipeline. Current Ultralytics YOLO handles unlabeled images during training.
